### PR TITLE
Rightstuff1

### PR DESCRIPTION
**Pull request by sharmoo**
adds padding as per Kevin's comments

### CODE REVIEW

File: client/components/PlaylistItem.jsx [hunk 1]
```diff
@@ -5,32 +5,32 @@ import styles from './RelSongsBox.css';
...
         <span className="gray">
           <FontAwesomeIcon icon="play" className="icon" />
           {' '}
-          {playlist.plays}
+          <span className="details">{playlist.plays}</span>
           {' '}
         </span>
         <span className="gray">
           <FontAwesomeIcon icon="heart" className="icon" />
           {' '}
```
Comment (kevinbece5) [inline client/components/PlaylistItem.jsx:20]: can you add that here as well to keep everything consistent?

File: client/components/RelSongs.jsx [hunk 2]
```diff
@@ -6,33 +6,25 @@ import styles from './RelSongsBox.css';
...
+      <div className="name">{song.artist}</div>
       <div className="songName">{song.songName}</div>
       <div><img alt={`${song.songName} albumPic`} src={`${song.albumPic}`} /></div>
       <div>
         <span className="gray">
           <FontAwesomeIcon icon="play" className="icon" />
-          {' '}
-          {song.plays}
-          {' '}
+          <span className="details">{song.plays}</span>
```
Comment (kevinbece5) [inline client/components/RelSongs.jsx:15]: Perfect!! 🔥

**Review by kevinbece5 — CHANGES_REQUESTED** [review]

File: client/components/RepostsBox.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      userReposts: fakeUserRepostsData,
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <Epony name="reposts" />
+        <ul>
+          {this.state.userReposts.map((user, key) => (
```
Comment (pon-y) [inline client/components/RepostsBox.jsx:21]: Could destructure userReposts to make cleaner

File: client/components/LikesBox.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      userlikes: fakeUserLikesData,
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <Epony name="likes" />
+        <ul>
+          {this.state.userlikes.map((user, key) => (
```
Comment (pon-y) [inline client/components/LikesBox.jsx:21]: Could destructure userLikes

**Pull request merged**